client/my-sites/plans/jetpack-plans/product-store/style.scss

```diff
@@ -13,13 +13,16 @@
 		margin: 0;
 	}
 
-	.button {
-		color: var( --color-neutral-100 );
-		border: 1px solid var( --color-neutral-100 );
+	a.button {
 		font-family: 'SF Pro Text', $sans;
 		font-size: $font-body;
 		font-weight: 600;
 	}
+
+	.button:not([disabled], :disabled, .is-primary) {
+		color: var( --color-neutral-100 );
+		border: 1px solid var( --color-neutral-100 );
+	}
```

Comment:
`color` and `border` was applied to all the buttons inside the product store which created problems for theme-based buttons especially when they are disabled.